CHANGES_NEXT_RELEASE

```diff
@@ -7,3 +7,6 @@
        a consequence, the 'attributeFormat' URI parameter is no longer needed in the 'GET /v1/contextTypes', so it is ignored.
 - Fix: DELETE updates are resolved locally, previously they were forwarded to CPrs (Issue #755)
 - Fix: The URI parameter 'notifyFormat' is now treated the same way for subscriptions as for registrations.
+- Add: Notifications and forwarded queries/updates are now controlled by a timeout. (Issue #880)
+       The default timeout is set to 5000 milliseconds (5 seconds), including connection, writing the request and reading the response.
+       To change this timeout, a new CLI option '-appTmo' has been added to the broker.
```

Comment:
I think it would be better (i.e. more easy for users) a parameter with "timeout" in the name, e.g. "-notifyTimeout".

In additio, to make thins more homogeneous, I would suggest to do some changes in the current "-timeout" CLI parameters (used for replica set timeout):

* Change the name to `-dbTimeout`
* Change its magnitude to and integer meaning miliseconds (currently `-timeout` use float meaning seconds, accepting decimal values).

The above changes involve a slight modificiation in exsting CLI. Howeve, taking into account that replica set are not widely used and that `-timeout` hasn't been here from a long time (it was introduced in the very previous Orion version, i.e. 0.20.0) I think the impact is controlled.